external_parser/unit_tests/test_log_converter.cc

```diff
@@ -134,3 +138,19 @@ BOOST_AUTO_TEST_CASE(ccb_payload_with_slot_id) {
 }
 
 BOOST_AUTO_TEST_SUITE_END()
+
+BOOST_AUTO_TEST_SUITE(log_converter_ca_format)
+BOOST_AUTO_TEST_CASE(ca_deferred_action_without_activation) {
+  std::string infile_path = "skip_learn/ca/deferred_action_without_activation.fb";
```

Comment:
q: is this joined log has skip learn enabled? The dsjson schema doesn't include a "_skip_learn" field like the cb and ccb schema, which affects cfe.

Reply:
Yes the joined log was generated with `--deferred_action` for the interaction and without `action_taken` in the observation, so skip_learn should be enabled.  Perhaps we should add a _skip_learn field to the dsjson schema for ca. The CaEvent schema includes `deferred_action` and is/set_skip_learn() is defined for all joined events. @olgavrou do you have thoughts?